code/modules/uplink/uplink_items.dm

```diff
@@ -2035,3 +2035,16 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 1
 	purchasable_from = UPLINK_CLOWN_OPS
 	illegal_tech = FALSE
+
+// Special equipment (Dynamically fills in uplink component)
+/datum/uplink_item/special_equipment
```

Comment:
Have we confirmed that, unless you're in the specific case where you can't get an item, this won't show?

Reply:
Yes, I have tested it on a local branch.